standard_reports/encounters/latest/report-manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "is_custom": false,
-  "version": "2.6.2",
+  "version": "2.6.3",
```

Comment:
Spoke with Ferg, sounds like this was a case of #7322 - this report was bumped to 2.6.2 from 2 different branches, first was extism 2.6.2, so boa 2.6.2 bump didn't get applied to existing DBs.